PubSub/tests/System/ManageSubscriptionsTest.php

```diff
@@ -264,6 +264,66 @@ public function testDeadLetterPolicy($client)
         sleep(2);
         $msg = $sub->pull();
         $this->assertEquals(1, $msg[0]->deliveryAttempt());
+
+        $sub->update([], [
+            'updateMask' => ['deadLetterPolicy']
+        ]);
+
+        $this->assertArrayNotHasKey('deadLetterPolicy', $sub->info());
+    }
+
+    /**
+     * @dataProvider clientProvider
+     */
+    public function testRetryPolicy($client)
```

Comment:
Can we add a test that unsets retry policy as well as one that resets retry policy to default?

Reply:
This testcase includes checking the unset behavior [here](https://github.com/googleapis/google-cloud-php/pull/3031/files#diff-33b589656543bbe2d634370fa5dfdaa6R322-R326). Could you explain a bit the distinction between unsetting and resetting to default?

Reply:
Oh I missed that. Unsetting test seems fine.

For the resetting to default part, users will pass retry_policy as RetryPolicy(). For unsetting, users will pass retry_policy as null.